src/decorator/live-decorator.ts

```diff
@@ -31,8 +31,8 @@ export class KalturaLiveEngineDecorator
     );
   }
 
-  private _handleError = (e: any) => {
-    this._plugin.reloadMedia = true;
+  private _handleError = () => {
+    this._plugin!.reloadMedia = true;
```

Comment:
I just thought about this, `this._hadError` should set for critical error only cause if we have another error like 'bufferStall' (recoverable) we don't want to active the decorator

Reply:
Actually I think reloadMedia here will cause problems in chrome

Reply:
I don't think we can do it in this way because initially plugin get critical error:
```
{
  "severity":1,
  "category":1,
  "code":1002,"
}
```
After investigation we discovered that actually issue happens because of invalid state of broadcast that comes from BE

Reply:
@semarche-kaltura I mean you can get multiple errors that are not critical we don't want to active the decorator for those errors - let's add the condition before we changing the hasError

Reply:
The second comment related to the investigation does it work on chrome when we activate the reload media on error?

Reply:
yes, it works
We have issue only in case, when BE respond with incorrect state for stream.